### ISSUE TITLE
[Link]: [The contrast ratio is wrong, when it is disabled status]

### ISSUE THREAD

**Issue by hxempty** [SAP/ui5-webcomponents#9599]
### Describe the bug

When I  use  axe scan the page ,we can see  error message.
![image](https://github.com/user-attachments/assets/6a1ef22e-279a-48aa-b738-6ad0c206441d)
The contrast ratio is wrong, when link component is disabled status

### UI5 Web Components for React Version

1.26.1

### UI5 Web Components Version

1.23.1

### Browser

Chrome

### Organization

SF

### Declaration

- [X] I’m not disclosing any internal or sensitive information.

**Comment by ivoplashkov** [comment]
Hello colleagues,

Issue could be reproduced in the new playground aswell - https://sap.github.io/ui5-webcomponents/nightly/components/Link/. Disabled link seems to have insufficient contrast with the sample background.
Could you please have a look?

Best regards,
Ivaylo

**Comment by hxempty** [comment]
> Ivaylo

Hi Ivaylo，I have check the page.It still show the issue
![image](https://github.com/user-attachments/assets/775bbc31-7019-4440-a898-473a23bd2c2b)

**Comment by didip1000** [comment]
Hey @hxempty,

As [W3C states](https://www.w3.org/TR/WCAG22/#contrast-minimum), contrast ratio isn't measured for disabled (inactive) elements:
> Success Criterion 1.4.3 Contrast (Minimum):
The visual presentation of [text](https://www.w3.org/TR/WCAG22/#dfn-text) and [images of text](https://www.w3.org/TR/WCAG22/#dfn-images-of-text) has a [contrast ratio](https://www.w3.org/TR/WCAG22/#dfn-contrast-ratio) of at least 4.5:1, **except for the following**:
...
> **Incidental**
Text or images of text that are part of an **inactive** [user interface component](https://www.w3.org/TR/WCAG22/#dfn-user-interface-components), that are [pure decoration](https://www.w3.org/TR/WCAG22/#dfn-pure-decoration), that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
...

Best,
Diana
